pkg/services/postgresqldb/postgresql.go

```diff
@@ -23,9 +37,20 @@ func New(
 	armDeployer arm.Deployer,
 	checkNameAvailabilityClient postgresSDK.CheckNameAvailabilityClient,
 	serversClient postgresSDK.ServersClient,
+	databasesClient postgresSDK.DatabasesClient,
```

Comment:
Why does the dbms-only manager need the database client?

Reply:
In fact, it doesn't look like you ever set a value for this attribute farther below when you initialize this manager. I'm guessing this attribute is extraneous.

Reply:
I think this is how the diff is rendering it. It's passed into the constructor like thing and used by the *db* service manager because it does need it. 

```
func New(
	armDeployer arm.Deployer,
	checkNameAvailabilityClient postgresSDK.CheckNameAvailabilityClient,
	serversClient postgresSDK.ServersClient,
	databasesClient postgresSDK.DatabasesClient,
) service.Module {
	return &module{
		allInOneManager: &allInOneManager{
			armDeployer:                 armDeployer,
			checkNameAvailabilityClient: checkNameAvailabilityClient,
			serversClient:               serversClient,
		},
		dbOnlyManager: &dbOnlyManager{
			armDeployer:                 armDeployer,
			checkNameAvailabilityClient: checkNameAvailabilityClient,
			databasesClient:             databasesClient,
		},
		dbmsOnlyManager: &dbmsOnlyManager{
			armDeployer:                 armDeployer,
			checkNameAvailabilityClient: checkNameAvailabilityClient,
			serversClient:               serversClient,
		},
	}
}
```

Reply:
Yep. You're right. I "unfolded" the lines above for a better point of reference and you're right. I'm looking at the args list for the module's "constructor." Got it.